code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -46,6 +46,7 @@
 	var/mob/living/simple_animal/bot/Bot
 	var/tracking = FALSE //this is 1 if the AI is currently tracking somebody, but the track has not yet been completed.
 	var/datum/effect_system/spark_spread/spark_system //So they can initialize sparks whenever/N
+	var/list/traits = list(TRAIT_CONSTRUCTION, TRAIT_CHEMISTRY, TRAIT_SURGICAL_EXPERT)
```

Comment:
This is pointless since the AI cant use an RCD or any construction / surgical tools

Reply:
Would this affect shells?

Reply:
no